simphony/cuds/simulation.py

```diff
@@ -14,8 +14,17 @@ class Simulation(object):
         Name of the underlying engine.
     engine_interface: engine.EngineInterface
         The interface to the engine, internal or fileio.
+    name: str
+        A custom name for this simulation
+    description: str
+        More information about this simulation
     """
-    def __init__(self, cuds, engine_name, engine_interface=None):
+    def __init__(self, cuds, engine_name, engine_interface=None,
+                 name=None,
+                 description=None):
+        self.name = name
+        self.description = description
+
```

Comment:
They are optional at the moment, exactly like the meta classes.  Perhaps we have to turn `CUDS` and `Simulation` classes into `CUDSItem` derivaties, to have one universal data structure to deal with (it has been always the aim). However, for now I just decided to keep it simple.